src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -1,20 +1,30 @@
 package billgates.view.gui;
 
 import javax.swing.*;
+import javax.swing.event.MenuEvent;
+import javax.swing.event.MenuListener;
+import javax.swing.filechooser.FileNameExtensionFilter;
 import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.io.File;
 
-public class TopMenuBar extends JMenuBar {
+public class TopMenuBar extends JMenuBar{
 
     public static final int DEFAULT_HEIGHT = 23;
     private final BoxLayout layout = new BoxLayout(this, BoxLayout.X_AXIS);
     private final JMenu importMenu = new GeneralMenu("Import");
     private final JMenu settingsMenu = new GeneralMenu("Settings");
     private final JMenu helpMenu = new GeneralMenu("Help");
+    private final Color[] colors = new Color[]{new Color(240, 140, 170), Color.RED, Color.ORANGE,
+            Color.YELLOW, Color.GREEN, Color.CYAN, Color.BLUE, Color.PINK, Color.WHITE};
+    private final String[] fonts = new String[]{"Modern No. 20", "Times New Roman", "Helvetica", "Arial", "Impact",
+            "Verdana", "Century", "Tahoma", "Copperplate"};
+    private MainFrame mainFrame;
```

Comment:
It seems like this is a `final` variable. The `mainFrame` instance would never change in the lifetime of our software.